README.md

```diff
@@ -55,5 +58,5 @@ NOMATCH https://1.1.1.2 19532
 Install golang, then run:
 
 ```
-go install github.com/hakluke/hakoriginfinder@latest
+go install github.com/carlospolop-forks/hakoriginfinder@latest
```

Comment:
It looks like you've changed the instructions to install it from your fork which is not ideal